examples/jupyter/nuclear-data-resonance-covariance.ipynb

```diff
@@ -0,0 +1,374 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x14c9e4583668>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.imshow(covariance)\n",
+    "plt.colorbar()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Another capability of the covariance module is selecting a subset of the resonance parameters and the corresponding subset of the covariance matrix. We can do this by specifying the value we want to discriminate and the bounds within one energy region. Selecting only resonances with J=2:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "    energy    J  neutronWidth  captureWidth  fissionWidthA  fissionWidthB  L\n",
+      "0   0.0314  2.0      0.000474        0.1072            0.0            0.0  0\n",
+      "1   2.8250  2.0      0.000345        0.0970            0.0            0.0  0\n",
+      "3  16.7700  2.0      0.012800        0.0805            0.0            0.0  0\n",
+      "4  20.5600  2.0      0.011360        0.0880            0.0            0.0  0\n",
+      "5  21.6500  2.0      0.000376        0.1140            0.0            0.0  0\n"
+     ]
+    }
+   ],
+   "source": [
+    "lower_bound = 2; #inclusive\n",
+    "upper_bound = 2; #inclusive\n",
+    "gd157_endf.resonance_covariance.ranges[0].res_subset('J',[lower_bound,upper_bound])\n",
+    "subset_first_five = gd157_endf.resonance_covariance.ranges[0].parameters_subset[:5]\n",
+    "print(subset_first_five)"
```

Comment:
As before, no need for variable assignment and explicit `print` -- the last expression in a cell will get printed by default